docs/design/Colors.stories.mdx

```diff
@@ -291,6 +269,116 @@ caution, it’s _bright!_
 <br />
 <br />
 
-## Swatch list
+## Base colors
+
+This is the underlying color palette that powers our semantic system.
+
+Direct use of these colors should be avoided, as they do not respond to theming
+or systemic changes.
 
-<ColorSwatches colors={colors} />
```

Comment:
This fixes the "undefined" and unexpected groupings. It's less elegant than automatically pulling all values, but
- it works
- it did not take me very long
- we don't frequently update the base palette 

So I am advocating that if an engineer is interested, they tackle this in a followup

![image](https://github.com/user-attachments/assets/c560dfa9-4dfc-4787-babc-15df3216f84d)